checkexperiments.py

```diff
@@ -0,0 +1,266 @@
+"""\
+Run status check on experiments.
+
+Example.
+
+    %(prog)s --username ACCESS_KEY_ID --password SECRET_ACCESS_KEY \\
+        --out experiments_statuses.log https://www.encodeproject.org
+"""
+import datetime
+import json
+import sys
+import copy
+import os.path
+import subprocess
+from urllib.parse import urljoin
+import requests
+from slackclient import SlackClient
+
+EPILOG = __doc__
+
+def run(out, err, url, username, password, search_query, accessions_list=None, bot_token=None, dry_run=False):
+    session = requests.Session()
+    session.auth = (username, password)
+    session.headers['Accept'] = 'application/json'
+
+    dr = ""
+    if dry_run:
+        dr = "-- Dry Run"
+
+    version = '0.11'
+
+    try:
+        ip_output = subprocess.check_output(
+            ['hostname'], stderr=subprocess.STDOUT).strip()
+        ip = ip_output.decode(errors='replace').rstrip('\n')
+    except subprocess.CalledProcessError as e:
+        ip = ''
+
+    initiating_run = 'STARTING Checkexperiments version ' + \
```

Comment:
We should mix format and string concatenation.   The whole thing should just be put in a string with a format function.
```
    initiating_run = (
        'STARTING Checkexperiments version '
        '{} ({}) ({}): {} on {} at {}'.format(
            version, url, search_query, dr, ip, datetime.datetime.now()
        )
    )
```

Reply:
done